proyecto-frontend/src/app/modules/product/component/product/product.component.html

```diff
@@ -0,0 +1,320 @@
+<div class="product-container">
+  <!-- Navbar de Productos -->
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container-fluid">
+      <button
+        class="navbar-toggler"
+        type="button"
+        data-bs-toggle="collapse"
+        data-bs-target="#navbarNav"
+        aria-controls="navbarNav"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="navbar-items collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="/secured"
+              >Categorias</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/product">Productos</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <!-- Tabla de Productos -->
+  <table class="table">
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Producto</th>
+        <th>GTIN</th>
+        <th>Precio</th>
+        <th>Categoría</th>
+        <th>Status</th>
+        <th>Acciones</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let product of products">
+        <td>{{ product.product_id }}</td>
+        <td>{{ product.product }}</td>
+        <td>{{ product.gtin }}</td>
+        <td>{{ product.price }}</td>
+        <td>{{ product.category_id }}</td>
```

Comment:
No nos interesa ver el id de la categoría
```suggestion
        <td>{{ product.category }}</td>
```